tool-plugins/intellij/src/main/resources/fileTemplates/diagram/lang-client.hbs

```diff
@@ -0,0 +1,15 @@
+function getLangClient() {
+    return {
+        isInitialized: true,
+        getAST: (params) => {
+            return new Promise((resolve, reject) => {
+                resolve({ ast: {{{ast}}}, parseSuccess: {{{parseSuccess}}} });
+            })
+        },
+        getEndpoints: () => {
+            return new Promise((resolve, reject) => {
+                resolve({});
+            })
+        }
+    }
+}
```

Comment:
Shall we add a newline? Some below files needs to be updated as well.

Reply:
Added a newline.